src/util/tmpl/fd_treap.c

```diff
@@ -205,12 +205,13 @@
      myele_t const * mytreap_ele_query_const( mytreap_t const * treap, char const * q, myele_t const * ele );
```

Comment:
As with the heap code, the meaning of the ele argument is unclear. Are you assuming all possible elements come from a linear pool?

Reply:
See global comment below.